### ISSUE TITLE
Error when price has 1 decimal digit

### ISSUE THREAD

**Issue by thiagohk** [vuejs-tips/v-money#81]
When the price is 50.1, for example, is being formatted as 5,01.
Running example https://jsfiddle.net/znewk1jb/

**Comment by ahoiroman** [comment]
Yes - in my case I have to use `number_format($this->price ,2)` in my API resource (PHP) to get this working. It's a ugly workaround, would love to see a fix :-)

**Comment by LeticiaLCavalcanti** [comment]
same problem here, '895' comes wrong 8.95! I don't know how can I fix it... Or how can I formatter a v-model money value using other way...

**Comment by hawkapparel** [comment]
For "fix" this problem you can use this:
let numberValue = 5200;
(parseFloat(numberValue)).toFixed(2);

will show: 5,200.00

**Comment by jonathanpmartins** [comment]
hey guys,
Unfortunately I don't think support will come soon. This repo seems to be abandoned! I decided to upgrade this package to be used with Vue3.

Welcome [`v-money3`](https://www.npmjs.com/package/v-money3)!

I'm interested to maintain `v-money3` for **vue 3** and beyond. Feel free to open this same issue [there](https://github.com/jonathanpmartins/v-money3/issues/new) if you are planning to upgrade your project.
